src/components/browser-banner/_macro.spec.js

```diff
@@ -5,83 +5,84 @@ import * as cheerio from 'cheerio';
+    });
 
-  it('has expected default content', () => {
-    const $ = cheerio.load(renderComponent('browser-banner', EXAMPLE_BROWSER_BANNER_MINIMAL));
+    it('has the default `bannerLeadingText`', () => {
+      const $ = cheerio.load(renderComponent('browser-banner', params));
 
-    const htmlContent = $('.ons-browser-banner__content')
-      .html()
-      .trim();
-    expect(htmlContent).toBe(
-      '<span class="ons-browser-banner__lead">This website no longer supports your browser.</span> You can <a class="ons-browser-banner__link" href="https://www.ons.gov.uk/help/browsers">upgrade your browser to the latest version</a>.',
-    );
-  });
+      const bannerLeadingText = $('.ons-browser-banner__lead')
+        .text()
+        .trim();
+      expect(bannerLeadingText).toBe('This website no longer supports your browser.');
+    });
 
-  it('has expected english content', () => {
-    const $ = cheerio.load(
-      renderComponent('browser-banner', {
-        ...EXAMPLE_BROWSER_BANNER_MINIMAL,
-        language: {
-          languages: [
-            {
-              ISOCode: 'en',
-              text: 'English',
-              current: true,
-            },
-          ],
-        },
-      }),
-    );
-
-    const htmlContent = $('.ons-browser-banner__content')
-      .html()
-      .trim();
-    expect(htmlContent).toBe(
-      '<span class="ons-browser-banner__lead">This website no longer supports your browser.</span> You can <a class="ons-browser-banner__link" href="https://www.ons.gov.uk/help/browsers">upgrade your browser to the latest version</a>.',
-    );
-  });
+    it('has the default `bannerCTA`', () => {
+      const $ = cheerio.load(renderComponent('browser-banner', params));
+
+      const bannerCtaHtml = $('.ons-browser-banner__cta')
+        .text()
+        .trim();
+      expect(bannerCtaHtml).toBe('You can upgrade your browser to the latest version.');
+    });
+
+    it('has the default `bannerLinkUrl`', () => {
+      const $ = cheerio.load(renderComponent('browser-banner', params));
+
+      expect($('.ons-browser-banner__link').attr('href')).toBe('https://www.ons.gov.uk/help/browsers');
+    });
 
-  it('has expected welsh content', () => {
-    const $ = cheerio.load(
-      renderComponent('browser-banner', {
-        ...EXAMPLE_BROWSER_BANNER_MINIMAL,
-        language: {
-          languages: [
-            {
-              ISOCode: 'cy',
-              text: 'Cymraeg',
-              current: true,
-            },
-          ],
-        },
-      }),
-    );
-
-    const htmlContent = $('.ons-browser-banner__content')
-      .html()
-      .trim();
-    expect(htmlContent).toBe(
-      '<span class="ons-browser-banner__lead">Nid yw’r wefan hon yn cefnogi eich porwr mwyach.</span> Gallwch <a class="ons-browser-banner__link" href="https://cy.ons.gov.uk/help/browsers">ddiweddaru eich porwr i’r fersiwn ddiweddaraf</a>.',
-    );
+    it('has `container--wide` class when `wide` is true', () => {
```

Comment:
This test occurs for both "no parameters provided" (unexpected) and "all parameters provided".